common/ruletypes.h

```diff
@@ -890,7 +890,7 @@ RULE_INT(Expansion, AutoGrantAAExpansion, -1, "Expansion to auto grant AAs up to
 RULE_CATEGORY_END()
 
 RULE_CATEGORY(Instances)
-RULE_INT(Instances, ReservedInstances, 30, "Number of instance IDs which are reserved for globals. This value should not be changed while a server is running")
+RULE_INT(Instances, ReservedInstances, 100, "Number of instance IDs which are reserved for globals. This value should not be changed while a server is running")
```

Comment:
I'm not sure what you mean here. I read it a few times

Reply:
FWIW these reserved instances are only really going to be used from fresh pulls of PEQ (new installs) - people are not going to be just pulling database updates for this to be as relevant as it is on PEQ for those looking to start taking advantage of the expansions content which is currently rapidly in development.

Our PEQ dump only includes

```cpp
	if (IsDumpStaticInstanceData()) {
		tables_to_dump += "instance_list";
		options += " --no-create-info --where=\"instance_list.is_global > 0 and instance_list.never_expires > 0\"";
	}
```

Reply:
This was just an observation that if someone manually changed an instance id to global/reserved it might prevent any tables that hold weak foreign keys to those instance ids from ever getting purged. Not really important for this a PR, more of a self note.